docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -670,49 +670,58 @@ paddlepaddle>=2.1
 
 | Operator Name in PaddlePaddle| Limitations|
 | :----------| :----------|
-| adpative_pool2d | The `NHWC` data_layout is not supported. |
```

Comment:
adaptive_pool2d is a paddle api, though it is saved with pool2d. I suggest to keep it.